web/src/scripts/utils/ElementTreeBuilder.js

```diff
@@ -143,6 +144,51 @@ export default class {
     }
   }
 
+  static buildObjectProp(attribute) {
+    const {value: {expression}} = attribute
+    const {type, loc: {start, end}, properties} = expression
+    const inferAstType = (value) => {
+      const typeOfValue = typeof value
+      switch (typeOfValue) {
+        case "number": {
+          return AST_TYPES.NumericLiteral
+        }
+        case "string": {
+          return AST_TYPES.StringLiteral
+        }
+        case "boolean": {
+          return AST_TYPES.BooleanLiteral
+        }
+      }
+    }
+    const objectProperties = properties.map((prop) => {
```

Comment:
Let's figure out a better way to do this. Because the point of an AST is that you don't have to parse manually. I also hope to limit `template` to function/raw values - there isn't much of a reason to use it for primitive types.